x-pack/plugins/infra/public/components/asset_details/hooks/use_asset_details_url_state.ts

```diff
@@ -53,10 +53,10 @@ const FlyoutTabIdRT = rt.union([
   rt.literal(FlyoutTabIds.OSQUERY),
 ]);
 
-const HostFlyoutStateRT = rt.intersection([
+const AssetDetailsStateRT = rt.intersection([
   rt.type({
     itemId: rt.string,
```

Comment:
The `itemId` is only used in the `use_hosts_table`. It can be moved over to the `use_hosts_table` state. With that, the `asset_details_url_state` will contain only attributes relevant to the Asset Details component. Wdyt?

Reply:
Good point, will move it to the `useHostsTableUrlState`